charts/kvisor/templates/agent.yaml

```diff
@@ -89,6 +89,8 @@ spec:
                 name:  {{ include "kvisor.clickhouse.fullname" . }}
         {{- end }}
           env:
+            - name: GOMEMLIMIT
+              value: "100MiB" # Make GC more frequent once reaching this heap size.
```

Comment:
Should this be a dynamic value based on total memory? We can do something like this:

```yaml
        - name: GOMEMLIMIT
          valueFrom:
            resourceFieldRef:
              resource: limits.memory
              divisor: '1' # maybe 2?
```

https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml#L201-L205